lib/spack/spack/build_systems/oneapi.py

```diff
@@ -107,7 +107,7 @@ def install_component(self, installer_path):
             bash = Executable("bash")
 
             # Installer writes files in ~/intel set HOME so it goes to prefix
```

Comment:
```suggestion
            # Installer writes files in ~/intel set HOME so it goes to staging directory
```

It is strange that we haven't seen a problem before. I think the installer writes an "intel" directory into $HOME directory but it is possible it is writing something else. In any case, using a dedicated directory seems like a better approach.

Reply:
Indeed it's weird, but it may well be a property of the filesystem which has some... interesting behaviours.